packages/frontity/src/cli/create-package.ts

```diff
@@ -2,53 +2,53 @@ import ora from "ora";
 import chalk from "chalk";
 import { normalize } from "path";
 import { prompt, Question } from "inquirer";
-import createPackage from "../functions/create-package";
-import { errorLogger, isFrontityProjectRoot, isThemeNameValid } from "../utils";
-import { EventEmitter } from "events";
-import { Options } from "../functions/create-package/types";
+import createPackage from "../commands/create-package";
+import {
+  errorLogger,
+  isFrontityProjectRoot,
+  isThemeNameValid,
+  log
+} from "../utils";
+import { Options } from "../steps/create-package";
 
 //  Command:
-//    create-package [name] [--typescript]
```

Comment:
Typescript support here was supposed to be removed in an earlier PR #243 but it was incomplete